ui/tract-renderer/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
in order for this to not collide with other imported translations in OneSky this file needs to have a unique name, `strings.xml` will collide with the other strings.xml file in this app